src/Sushi.Mediakiwi/Presentation/Logic/Navigation.cs

```diff
@@ -1109,8 +1111,11 @@ public string NewBottomNavigation(Beta.GeneratedCms.Console container, ContentLi
                 {
                     if (container.CurrentListInstance.wim.Page.Body.Navigation.Menu.DeleteButtonTarget == ButtonTarget.BottomLeft || container.CurrentListInstance.wim.Page.Body.Navigation.Menu.DeleteButtonTarget == ButtonTarget.BottomRight)
                     {
+                        string classNames = string.IsNullOrEmpty(deleteRecord) ? "flaticon icon-trash-o" : "submit";
+                        string label = string.IsNullOrEmpty(deleteRecord) ? string.Empty : deleteRecord;

```

Comment:
This line can be removed and the `deleteRecord `property can be used directly, since you're setting it to string.empty anyways